src/components/Tabs/Tab/Tab.scss

```
@@ -14,19 +14,26 @@ $c-tabs-item-border-color: $color-primary !default;
 $c-tabs-item-hover-border-color: $color-gray-200 !default;
 
 $c-tabs-item-dark-color: $color-gray-100 !default;
-$c-tabs-item-dark-active-color: #FFF !default;
-$c-tabs-item-dark-border-color: #FFF !default;
+$c-tabs-item-dark-active-color: #fff !default;
+$c-tabs-item-dark-border-color: #fff !default;
 $c-tabs-item-dark-hover-border-color: $color-gray-25 !default;
 
 /* Component
    ========================================================================== */
 
 .c-tab-item {
 	color: $c-tabs-item-color;
-	padding: 1.2rem;
 	font-weight: 500;
 	transition: color 0.3s ease-in-out;
 
+	&:first-child {
+		padding: 1.2rem 1.7rem 1.2rem 1.2rem;
+	}
+
+	&:not(:first-child) {
+		padding: 1.2rem 1.7rem;
+	}
+
```

if you put first child after the global rule it will win. So no need to explicitly select the "not first child"
```suggestion
	padding: 1.2rem 1.7rem;

	&:first-child {
		padding: 1.2rem 1.7rem 1.2rem 1.2rem;
	}

```